sdk/communication/azure-communication-common/src/main/java/com/azure/android/communication/common/CommunicationTokenRefreshOptions.java

```diff
@@ -47,15 +47,12 @@ public CommunicationTokenRefreshOptions(Callable<String> tokenRefresher, boolean
      *                           tokenRefresher before token expiry by minutes set
      *                           with setCallbackOffsetMinutes or default value of
      *                           two minutes
-     * @param token the serialized JWT token, cannot be null
+     * @param initialToken the serialized JWT token, cannot be null
      */
     public CommunicationTokenRefreshOptions(Callable<String> tokenRefresher, boolean refreshProactively, String initialToken) {
         if (tokenRefresher == null) {
             throw new IllegalArgumentException("Missing required parameters 'tokenRefresher'.");
         }
-        if (initialToken == null) {
-            throw new IllegalArgumentException("Missing required parameters 'initialToken'.");
-        }
```

Comment:
Please keep this. If initialToken is not available, caller should be using the other constructor